LiteCore/Support/LogEncoder.cc

```diff
@@ -81,7 +82,8 @@ namespace litecore {
         return int64_t(_st.elapsed() * kTicksPerSec);
     }
 
-    void LogEncoder::vlog(const char *domain, ObjectRef object, const char *format, va_list args) {
+    void LogEncoder::vlog(const char *domain, const map<unsigned, string> &objectMap,
+        ObjectRef object, const char *format, va_list args) {
```

Comment:
If you have a chance could you indent this to line up with the `(` on the prev line?